src/coord/src/coord.rs

```diff
@@ -4499,7 +4500,7 @@ pub async fn serve(
                 logging,
                 internal_cmd_tx,
                 metric_scraper,
-                global_timeline: timeline::TimestampOracle::new(now()),
+                global_timeline: timeline::TimestampOracle::new(now(), move || (now.deref())()),
```

Comment:
Did this not work? I know its ugly, but its a more common way to do this 
```suggestion
                global_timeline: timeline::TimestampOracle::new(now(), move || (&*)()),
```

Reply:
Wait what? Where'd now go? I typed it this way because...it didn't make clippy complain but am happy to do the above. Mostly this change is blocked on uh unknown problems, so might not go in. If it does I'll try some variation of the this.